core/src/main/resources/reference.conf

```diff
@@ -184,6 +184,8 @@ backend {
         #concurrent-job-limit = 5
 
         run-in-background = true
+        # Configures any logic which should run after the command completes
+        # script-epilogue = ""
```

Comment:
Very ToL: Would *personally* prefer if this comment showed the default rather than some-other-value (`""`).